### DISCUSSION TITLE
How do you view motion detection thumbnails/previews in the review page?

### DISCUSSION THREAD

**Discussion by mgkeirug** [blakeblackshear/frigate#15512] [Ask A Question]
Maybe I'm missing something but when I go to the review page and click on alerts, the page is populated with thumbnails/previews from _all_ of the alerts from that day which I can conveniently click on to watch.  However, when I click on motion, it only shows one thumbnail/preview, the most recent one.  The only way to review all of the motion recordings from that day is to scroll through the timeline which is tedious and very inconvenient.  I want to be able to see the motion thumbnails on the preview page the same way you can see the alerts/events without enabling 'detect'.

**Comment by hawkeye217** [comment]
The motion review page is not showing individual moments throughout the day. It shows all of your cameras. The timeline then can be used to scrub throughout the last 24 hours where the yellow bars show where motion was detected. As you scrub, yellow outline will also appear around the cameras where motion was detected as well.
